### DISCUSSION TITLE
Question about MAGMA Core

### DISCUSSION THREAD

**Discussion by tayloreisman16** [magma/magma#7501] [General]
Hey guys,

I was wondering if an AWS account is necessary. I'm installing the system, but we'd like to run Magma as contained on a single computer as possible. I've seen that the IRIS testbed requires access to JFED, but again, we would like to run it locally as much as possible. It is not an issue if it requires cloud access, just would appreciate some guidance.

**Comment by pramchan** [comment]
heck this ans see if you can do without AWS a/c. http://iristestbed.eu/index.php/fb-magma-core

**Comment by pramchan** [comment]
Check this and see if you can do without AWS a/c. 
http://iristestbed.eu/index.php/fb-magma-core
